print/assets/toc.css

```diff
@@ -0,0 +1,59 @@
+/* mostly copied from https://gitlab.coko.foundation/pagedjs/pagedjs-documentation/-/blob/master/Part09_Table-of-content.md */
+
+ul.toc {
+    list-style: none;
+}
+
+ul.toc .toc-element a::after {
+    content: target-counter(attr(href url), page);
+    float: right;
+}
+
+ul.toc .toc-element-level-1 {
+    margin-top: 25px;
+    font-weight: bold;
+}
+
+
+ul.toc .toc-element-level-2 {
+    margin-left: 25px;
+}
+
+/*  leader() doesn't work for now, replace with the hack bellow */
+
+/* ul.toc .toc-element a::after{
+    content: leader('.') " p. " target-counter(attr(href), page);
+    float: right;
+} */
+
+/* hack for leaders */
+
+/* stylelint-disable no-duplicate-selectors */
+ul.toc {
+    overflow-x: hidden;
+}
+
+ul.toc .toc-element::after {
+    content: ".................................................................................................................................................";
```

Comment:
... really? What about landscape / portrait? What if we ever change the font, font size or paper format? If you really want dots, use a dotted border, like e.g. here: https://stackoverflow.com/a/65186779

Reply:
https://github.com/ecamp/ecamp3/pull/2562/commits/fa18f1a854254959bc104ca602aafea20ff6b3e5